### DISCUSSION TITLE
docs/how-to-connect/saml/appgate/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2889] [Docs comments]
# docs/how-to-connect/saml/appgate/

How to Use Casdoor as SAML IdP for Appgate

https://casdoor.org/docs/how-to-connect/saml/appgate/

**Comment by tranhoangvu** [comment]
it's not working. i got the message from my Appgate Admin "Invalid SAML token. Please contact your administrator."
please re-check

**Reply by tranhoangvu** [reply]
i fixed it, Appgate doesn't support SAML Compression, need to disable it in Casdoor application edit page
